Runners/ConsoleApp/Program.cs

```diff
@@ -23,11 +24,14 @@ static void Main()
         void Main(IMessageSender messageSender)
         {
             var pluginExecutor = new PluginExecutor(messageSender, new MemoryDal(), new PluginsMenu(), new PluginsManager());
-
+            Console.WriteLine("Hi! I am Your amazing bot and I will be happy to be of service to you on several topics.");
+            Console.WriteLine("Please type the number of the plugin you would like to help, or type help to get the list of options.\r\nAt any point you can type MAIN or HOME to return to the main menu.");
             while (true)
             {
+                Console.Write("main{0}> ", level);
                 var msg = Console.ReadLine();
-                pluginExecutor.Run(msg, " ");
+                level=pluginExecutor.Run(msg, " ");
+               
```

Comment:
Please format the code
https://docs.microsoft.com/en-us/answers/questions/550115/automatic-format-code-in-visual-studio.html

Reply:
ok

Reply:
If you fixed this, please click on the "Resolve conversation"